DeAceroDLabs/marketplace-frontend
# feat: [DMP-510] Acciones en vista de Carrito

## What is done

[[DMP-510] Acciones en vista de Carrito](https://deacero.atlassian.net/browse/DMP-510)

## How is done

1. Added a link to the "ir a pagar" button to /carrito/pago
2.Added a link to the "agregar otro producto" button to /home 

## How to test

1. Run the project via `npm start`
3. Go to /carrito/productos
4. Click on "ir a pagar", you should be redirected to /carrito/pago
5. Click on "agregar otro producto", you should be redirected to the home page
